engine/src/cubos/engine/imgui/imgui.hpp

```diff
@@ -1,19 +1,23 @@
 /// @file
 /// @brief Functions for initializing and using ImGui.
-/// @ingroup core-ui
+/// @ingroup imgui-plugin
 
 #pragma once
 
 #include <cubos/core/gl/render_device.hpp>
 #include <cubos/core/io/window.hpp>
 
-namespace cubos::core::ui
+using Framebuffer = cubos::core::gl::Framebuffer;
+using Window = cubos::core::io::Window;
+using WindowEvent = cubos::core::io::WindowEvent;
```

Comment:
Although this is a 'private' header, I think we should follow the same standards as other headers and thus not use `using` anywhere here.